### PR TITLE
New organizations page - "What is the difference between a user accou…

### PR DESCRIPTION
**Pull request by dunjardl**
@pyromaniackeca please review

### CODE REVIEW

File: source/docs/organizations/what-is-the-difference-between-a-user-account-and-an-organization.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+On the other hand, **organizations** are perfect for businesses and teams which
+have the need to be able to allow more than one user to manage their projects,
+or who would like to be able to give different permissions to different teams
+and/or collaborators.
+
+## Next steps
+
+If you are interested in creating an organization, here are the next steps
+you can take:
+- [Set up an organization](docs/organizations/setting-up-an-organization.html)
```
Comment (pyromaniackeca) [inline source/docs/organizations/what-is-the-difference-between-a-user-account-and-an-organization.md]: Let's not link any of these steps until we have the pages to link all of them.

Comment (dunjardl) [inline source/docs/organizations/what-is-the-difference-between-a-user-account-and-an-organization.md]: Agreed.

**Comment by pyromaniackeca** [comment]
I think it's good. I would also try and add something like A user is an account representing a person, while an organization is a group of users that one or more users can own. I'm guessing you can write this in a better way, what do you think?

**Comment by dunjardl** [comment]
:+1: I like that explanations, it's much clearer. I'll update the doc to make these two changes.

File: source/docs/organizations/what-is-the-difference-between-a-user-account-and-an-organization.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+A **user account** allows you to create projects and invite collaborators. On
+the other hand, **organizations** are used by businesses and teams which
+have the need to be able to allow more than one user to manage their projects,
+or to set different permission levels for different teams and/or collaborators.
+
+## Next steps
+
+If you are interested in creating an organization, here are the next steps
+you can take:
+- Set up an organization
```
Comment (markoa) [inline source/docs/organizations/what-is-the-difference-between-a-user-account-and-an-organization.md]: This one could be a link, and the other two after the pages have been created?

Comment (pyromaniackeca) [inline source/docs/organizations/what-is-the-difference-between-a-user-account-and-an-organization.md]: @markoa We discussed this with @darkofabijan he said we shouldn't link a step if we can't link all of them. What's the final verdict on that?

Comment (markoa) [inline source/docs/organizations/what-is-the-difference-between-a-user-account-and-an-organization.md]: It's not very important right now, the number of pages will increase soon anyway. Feel free to merge.

**Pull request merged**